### DISCUSSION TITLE
Felhőalapú pénztárgép és vevői kijelző

### DISCUSSION THREAD

**Discussion by wlln22** [nav-gov-hu/eRECEIPT#39] [Kérdések és válaszok]
A készülő bizonylat adatainak megjelenítése a vevő részére az AE által vezérelt kijelzőn kötelező. Felhőalapú pénztárgép esetén ennek az elvárásnak hogyan lehet megfelelni? Ezen adatok megjeleníthetőek a készülék saját kijelzőjén? Pl.: mobiltelefon képernyőjén. Felhőalapú epg esetén nem lesz kötelező a külön vevői kijelző?

**Comment by BatariZ** [comment]
Az időközben megjelent rendelet (A nemzetgazdasági miniszter 8/2025. (III. 31.) NGM rendelete az e-pénztárgépek forgalmazásáról, üzemeltetéséről, valamint az e-pénztárgépek és az e-nyugta kiállításának követelményeiről), illetve az aktualizált Fejlesztői dokumentáció tartalmazza a válaszokat a feltett kérdésekre.
